consensus/polybft/state.go

```diff
@@ -840,6 +844,35 @@ func (s *State) bucketStats(bucketName []byte) *bolt.BucketStats {
 	return stats
 }
 
+// getProposerCalculatorSnapshot gets latest proposer calculator snapshot
+// we are keeping two snapshots just in case if latest write mess up snapshot
```

Comment:
Is this comment entirely correct? I don't see we are preserving two snapshots, since we are constantly storing a given instance by using the same value for `proposerCalcSnapshotKey`?